airbyte-integrations/connectors/source-dockerhub/unit_tests/test_streams.py

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from http import HTTPStatus
+from unittest.mock import MagicMock
+import json
+import requests
+import pytest
+from source_dockerhub.source import DockerHub
+
+
+@pytest.fixture
+def patch_base_class(mocker):
+    # Mock abstract methods to enable instantiating abstract class
+    mocker.patch.object(DockerHub, "path", "v0/example_endpoint")
+    mocker.patch.object(DockerHub, "primary_key", "test_primary_key")
+    mocker.patch.object(DockerHub, "__abstractmethods__", set())
```

Comment:
Do we need this? `DockerHub` isn't an abstract class so it should already be implementing any abstract methods from `HttpStream`